roles/openshift_master/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-- name: restart openshift-master
-  service: name=openshift-master state=restarted
+- name: restart master
+  service: name={{ product_type }}-master state=restarted
```

Comment:
@brenton @ashcrow handlers are processed differently than tasks, so I suspect that was the issue at play.  What would need to be done if separate naming was going to happen would be to have 2 separate handlers (one for each name) and use the variable in the notify parameter of the task. Since we are consolidating on naming, the point is really moot though.